package.json

```
@@ -251,13 +251,23 @@
         "command": "one.viewer.metadata.showFromOneExplorer",
         "title": "ONE: Show Metadata",
         "category": "ONE"
+      },
+      {
+        "command": "one.cfgEditor.setDefaultValues",
+        "title": "ONE: Set Default Values",
+        "category": "ONE"
       }
     ],
     "keybindings": [
       {
           "command": "one.explorer.runSingleSelectedCfg",
           "key": "ctrl+f7",
           "when": "OneExplorerView.active && one.explorer:hasSelectedCfg"
+      },
+      {
+        "command": "one.cfgEditor.setDefaultValues",
+        "key": "ctrl+shift+/",
+        "when": "one.CfgEditor:activeCfgIsVisible"
```

```suggestion
        "when": "activeCustomEditorId == one.editor.cfg"
```

You can use this context, it's given by vscode. :-D

---

Ok. Thank you.